package.json

```diff
@@ -22,9 +38,11 @@
     "ignore": "^5.2.4"
   },
   "devDependencies": {
+    "@arethetypeswrong/cli": "^0.16.4",
     "@babel/core": "^7.22.9",
     "@babel/eslint-parser": "^7.22.9",
     "@eslint-community/eslint-plugin-mysticatea": "^15.5.1",
+    "@types/eslint": "^9.6.1",
```

Comment:
Maybe it's best to shadow the eslint version here? (`^8`)

Reply:
We could update version of `eslint` instead since `eslint` now provides its own type definitions so no need to use `@types/eslint`. Thoughts?